src/com/mendix/widget/carousel/components/__tests__/Carousel.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { ReactWrapper, ShallowWrapper, mount, shallow } from "enzyme";
+import { DOM, createElement } from "react";
+
+import { Carousel, CarouselProps, Image } from "../Carousel";
+import { CarouselItem } from "../CarouselItem";
+
+describe("Carousel", () => {
+    let images: Image[];
+    describe("shallow component", () => {
```

Comment:
Had a feeling you were going to say that :)
Struggled with this for a bit. Was trying to avoid having to redeclare `carousel` & `carouselWrapper` for every `describe()`.